pkg/backingstore/backingstore.go

```diff
@@ -1088,7 +1087,6 @@ func MapSecretToBackingStores(secret types.NamespacedName) []reconcile.Request {
 			})
 		}
 	}
-	log.Infof("will reconcile these backingstores: %v", reqs)
```

Comment:
I think this is helpful information for us going over the logs... maybe we can just join it together with the first one? something like: 
`log.Infof("will reconcile these backingstores: %v due to changes in the secret %v", reqs, secret)`
WDYT?

Reply:
We should not log it, look at https://github.com/noobaa/noobaa-operator/issues/1282

Reply:
@jackyalbo @liranmauda  the issue of #1282 is not the fact that the message is logged. 2000 reconciles with a multitude of api requests with each reconcile is the issue. Currently, the noobaa ServiceAccount is the account with most API requests overall on each of our clusters with ODF. IMHO the operator should react on changes/events not constantly reconcile regardless of changes. Just my 2cents. ;-)

Reply:
@vinzent 
I will assume this, and correct me if I am wrong:
You are only aware of noobaa reconciliation as it was reported in the logs.
I guess that if you will add printing to other components you will see the same.

The reconcile is watching for changes, which means that it needs to be aware of when (or as close as can be) to the change. 

If you are using an upstream version on your upstream ODF (is there something like that?), you can add a lot more printing to the reconcilers of all components, build that, and replace the images. 

again I will assume that it will be the same.

P.S. If you are using a downstream ODF of RedHat/IBM, do not use upstream images... (just making sure)

Reply:
> @vinzent I will assume this, and correct me if I am wrong: You are only aware of noobaa reconciliation as it was reported in the logs. I guess that if you will add printing to other components you will see the same.

Of course there are reconciles with each operator. But I would expect that most of them don't do 2'000 each hour.

Some numbers from 2024-03-12 to compare:

* Noobaa: 1 ObjectBucketClaim, no changes all day long to this ObjectBucketClaim: 890'151 api requests, 1'813'784 lines of log
* Cert-Manager: 1243 Certificates managing 1243 secrets, 24 certs created/renewed: 16'169 api requests, 295 lines of log

(Cert-Manager is probably "hiding" api requests due to caching, where noobaa doesn't cache at all?)

> 
> If you are using an upstream version on your upstream ODF (is there something like that?), you can add a lot more printing to the reconcilers of all components, build that, and replace the images.
> again I will assume that it will be the same.
> 
> P.S. If you are using a downstream ODF of RedHat/IBM, do not use upstream images... (just making sure)

Of course, we are using official ODF images. We're facing some weird issues on 2 of our ODF/Noobaa installations. I just thought that reporting suspicious behavior upstream makes sense.

Reply:
@vinzent 
> Of course, we are using official ODF images. We're facing some weird issues on 2 of our ODF/Noobaa installations. I just thought that reporting suspicious behavior upstream makes sense.

Thank you for reporting upstream, it is appreciated. 
Upstream resolving an issue, does not guarantee which official version will be released with this fix.
Currently, our master is targeted to 5.16 (roughly equal to the 4.16 RedHat image).

Another Point is that Upstream suggestions do not consider officially supported flows. 

Any issue/question is always welcome.
I wanted to raise awareness for the differences between upstream and downstream 🙂